sql/structured.proto

```diff
@@ -177,6 +177,16 @@ message TableDescriptor {
   optional PrivilegeDescriptor privileges = 12;
   // Columns or indexes being added or deleted in a FIFO order.
   repeated DescriptorMutation mutations = 13 [(gogoproto.nullable) = false];
+  // The table leader lease. A single goroutine across a cockroach cluster
+  // can own it and as a result will execute scheduled schema changes for
+  // this table. 
+  message LeaderLease {
```

Comment:
Can we give this a more descriptive name than `LeaderLease`? The word "leader" is already overloaded between raft leadership and the `storage` range leader lease. Maybe `SchemaLease`?